readme.md

```diff
@@ -81,6 +81,13 @@ to start with “kernel documentation”, which is dense).
     Presentation by the author of eBPF.
 -   [Extending extended BPF](https://lwn.net/Articles/603983/)
     (Jonathan Corbet, July 2014).
+-   Greg Marsden made some documentions or notes about eBPF
+    -	[A Tour of Program Types](https://blogs.oracle.com/linux/notes-on-bpf-1) A description of all existing hooks for BPF program types, and of their interest.
+    -	[BPF helper functions](https://blogs.oracle.com/linux/notes-on-bpf-2) A review of the kernel functions that can be called from within eBPF programs
```

Comment:
The first two items are indented with a tab (after the dash). Please use spaces, and in the same amount, for all items.